workers/ohsome_quality_analyst/utils/helper.py

```diff
@@ -188,3 +188,15 @@ async def sem_task(task: Coroutine, semaphore=asyncio.Semaphore(4)) -> Coroutine
     """Run task with semaphore. Semaphore limits num of concurrent executions."""
     async with semaphore:
         return await task
+
+
+async def gather_with_semaphore(tasks: list) -> Coroutine:
+    """Limit the number of tasks executed at a time."""
+    # Semaphore needs to initiated inside of the event loop
+    semaphore = asyncio.Semaphore(4)
```

Comment:
Will this value (`4`) be configurable in the future? Maybe add a todo comment if not already addressed in an issue.

Reply:
It is already configurable in #255